Eastern-Research-Group/mywaterway
# Feature/404 319 documents not found

## Related Issues:
* [HMW-404](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=864&view=detail&selectedIssue=HMW-404)

## Main Changes:
* Removed the "Documents" row from the items in the "Clean Water Act Section 319 Projects" sub-tab of the _Restore_ tab.
* Removed the loading of the `logo.png` image file in the static HTML files. It seems that the imported EPA scripts take care of loading the logo, now.
    * _Prettier_'s formatting made it look like a lot of changes were made to `400.html` and `500.html`, but the only change made to both is the single change made to `index.html`.

## Steps To Test:
1. Navigate to http://localhost:3000/community/dc/restore, open a project item, and confirm that the _Documents_ row under _Project Details_ no longer exists.
2. Confirm that the EPA logo is still visible at the top of any page, and that it links to https://www.epa.gov.
3. Navigate to http://localhost:3000/communities, and verify the logo is intact and functional.
4. In `/server/app/server/routes/404.js`, change `400.html` to `500.html`, revisit http://localhost:3000/communities, and confirm the logo is still there.
